### PR TITLE
feat: SwiftLint support

### PR DESCRIPTION
**Pull request by zippy1978**
SwiftLint support

- [x] Rules repository JSON file generator (with rules, type and debt info)
- [x] Merge existing debt info from sonar-swift Sqale model
- [x] Integrate rules to builtin profile
- [x] SwiftLintSensor (run SwiftLint, parse results and record issues)
- [x] SwiftLint result parser
- [x] SwiftLint rules update script (added new rules, prompt user for manual input for missing data)

DoD
- [x] Unit tests
- [x] Documentation (and developer documentation) update

### CODE REVIEW

**Review by davidy4ng — APPROVED** [review]

File: DEVELOP.md [hunk 1]
```diff
@@ -37,3 +37,72 @@ When started locally SonarQube UI is available at http://localhost:9000
...
+Closure expressions should have a single space inside each brace.
+
+Severity ? (1 = BLOCKER, 2 = CRITICAL, 3 = MAJOR, 4 = MINOR, 5 = INFO)
+```
+
+## Adding an issue sensor
+
+The plugin comes with several issue sensor, that is a piece of code running on source files and reporting issues.
+
+For example, ``SwiftLintSenor`` is a sensor using SwiftLint tool to run.
```
Comment (gaelfoppolo) [inline DEVELOP.md:94]: `SwiftLintSenor` -> `SwiftLintSensor`

File: scripts/migrate.groovy [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+ */
+
+
+// FIXME : this is a temporary script to merge Sqale data with rules JSON file
+// Should be deleted when all linters will be migrated
+
+import groovy.json.JsonSlurper
+import groovy.json.JsonBuilder
+
+//print "What is your name? "
```
Comment (gaelfoppolo) [inline scripts/migrate.groovy:26]: Example code to remove?

File: README.md [hunk 3]
```diff
@@ -20,14 +20,14 @@ Let us know if you want to get involved.
...
-| Size                | IN PROGRESS | IN PROGRESS |
-| Issues              | IN PROGRESS | IN PROGRESS |
-| Tests               | IN PROGRESS | IN PROGRESS |
-| Coverage            | IN PROGRESS | IN PROGRESS |
-| Complexity          | IN PROGRESS | IN PROGRESS |
-| Syntax highlighting | IN PROGRESS | IN PROGRESS |
+| Feature             | Swift                                                 | Objective-C |
+|---------------------|-------------------------------------------------------|-------------|
+| Size                | IN PROGRESS                                           | IN PROGRESS |
+| Issues              | [SwiftLint](https://github.com/realm/SwiftLint) rules | IN PROGRESS |
```
Comment (gaelfoppolo) [inline README.md:26]: Maybe add a version? New version with new rules will not be taken into consideration but might be reported as issues by users. With a version number they could check if that rule existed and if not, maybe open a PR.

**Review by gaelfoppolo — CHANGES_REQUESTED** [review]

File: DEVELOP.md (hunk 1 above)
Comment (zippy1978) [reply DEVELOP.md:94]: Done !

File: README.md (hunk 3 above)
Comment (zippy1978) [reply README.md:26]: Done !

File: scripts/migrate.groovy (hunk 2 above)
Comment (zippy1978) [reply scripts/migrate.groovy:26]: Removed !
Anyway this script is temporary, and will be removed once OCLintSensor is done.

**Review by gaelfoppolo — APPROVED** [review]

**Pull request merged**